cli/src/pcluster/resources/imagebuilder/parallelcluster_validate.yaml

```diff
@@ -167,6 +167,36 @@ phases:
               VERSION=$(echo ${PATTERN} | tr -d '\n' | cut -d = -f 2 | xargs)
               echo ${VERSION}
 
+      - name: CudaSamplesSrcDir
+        action: ExecuteBash
+        inputs:
+          commands:
+            - |
+              set -v
+              cuda_ver="{{ validate.CudaVersion.outputs.stdout }}"
+              if [ ${cuda_ver} \> '11.4' ]; then
+                PATTERN=$(grep -F "default['cluster']['nvidia']['cuda_sample_version']" {{ CookbookDefaultFile }})
```

Comment:
I couldn't find `default['cluster']['nvidia']['cuda_sample_version']`.
Is this related to another PR? Can you please reference it into the description?

Reply:
That change is coming in an as yet to be generated PR for parallelcluster-cookbook. So technically, with this PR, cuda_ver will be empty, so the logic here still kind of works but is definitely sloppy. I will fix this.